packages/commonwealth/server/migrations/20231212140801-divastaking-community-and-contract-update.js

```diff
@@ -0,0 +1,759 @@
+'use strict';
+
+const { hasher } = require('node-object-hash');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction();
+    // Update "Communities" and "Contracts" tables as per your previous script
```

Comment:
Check for existence of known values(in production) and allow migration to pass if not in some other db(ie a clean local db)

Reply:
Checking for existence and passing through as needed